BroadcomCloudSwg/client/broadcom_cloud_swg_client.py

```diff
@@ -106,7 +107,10 @@ def get_real_time_log_data_url(
         """
         params: dict[str, int | str] = {
             "endDate": int(end_date.timestamp()) * 1000,
-            "startDate": int(start_date.timestamp()) * 1000,
+            # Try to convert date into YYYYMMDDHH format according to this
+            # https://techdocs.broadcom.com/us/en/symantec-security-software/web-and-network-security/cloud-swg/help/cloudswg-api-reference/report-sync-about.html
+            "startDate": start_date.astimezone(pytz.UTC).strftime("%Y%m%d%H"),
```

Comment:
According to [the documentation](https://techdocs.broadcom.com/us/en/symantec-security-software/web-and-network-security/cloud-swg/help/cloudswg-api-reference/report-sync-about/report-sync-api.html#topic.dita_4509cfa5-a73d-4ebc-8a12-9e398dc2e8f5_section_6), the dates must be epoch in millisecond. In your change, the `startDate` will be a string; that doesn't match the Broadcom documentation.